pmda/util.py

```diff
@@ -205,6 +208,36 @@ def second_order_moments(S1, S2):
         partition `T`, the mean `mu` and the "second moment" `M` (sum of
         squares) for the combined data.
 
+    Notes
+    -----
+    The variance and sum of squares, otherwise known as the first and second
+    order moments, respectively, are defined as
+
+    M_{1, S_{i}} = \sum_{t=t_{0}}^{T}(x_{t} - \bar{x})
```

Comment:
For display math
```rest
.. math::
   M_{1, S_{i}} = \sum_{t=t_{0}}^{T}(x_{t} - \bar{x})
```

and for inline math
```
A wonderful equation is :math:`e^{i\pi} + 1 = 0` --- everyone can agree on this.
```

Reply:
Build the docs and check that they display correctly.

Reply:
I've tried to build it using 'python setup.py build_sphinx' but I keep getting errors. The current error is: 

"/nfs/homes/nawtrey/Library/pmda/pmda/rmsf.py:docstring of pmda.rmsf.RMSF:6:Unexpected section title.

Parameters
----------"

For some reason it doesn't like my title "Parameters" in the RMSF class docstring. The formatting is identical to DensityAnalysis, so I'm not quite sure why this is an issue.

Reply:
These things can be tricky. Some ideas
- look at the sphinx napoleon extension docs http://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html – we use it to use numpy-style reST
- numpy style reST https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt
- debug by divide and conquer (or bisection): take out parts until it works, then put parts back in until you find the thing that makes it fail